src/main/index.js

```diff
@@ -470,6 +470,7 @@ const initialProjectDataPromise = (async () => {
         const projectData = await promisify(fs.readFile)(projectPath, null);
         return projectData;
     } catch (e) {
+        return;
```

Comment:
@alx-pi could you pop a note in on the context for this `return` statement please?